ranger/gui/widgets/statusbar.py

```diff
@@ -304,7 +304,10 @@ def _get_right_part(self, bar):  # pylint: disable=too-many-branches,too-many-st
             # away and don't see them anymore.
             right.add('Mrk', base, 'marked')
         elif target.files:
-            right.add(str(target.pointer + 1) + '/' + str(len(target.files)) + '  ', base)
+            if target.filter == None:
```

Comment:
Maybe we should simply always show the real total if it differs? Regardless of filtering.

Reply:
In this case we need compare last index or number of selected all files and number here.
In my case you just should compare numbers `x` and `z` in `y/x(z)` notation.
Anctually, `y/x(z)` it's not beatiful format, but I don't know how change it else.